Fixed: Incorrect Client SDK Sample

krist00fer opened the pull request:
The sample incorrectly created a Uri to a Storedprocedure named "sproc" that actually have the id "ValidateDocumentAge"

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

krist00fer committed:
Fixed: Incorrect Client SDK Sample

The sample incorrectly created a Uri to a Storedprocedure named "sproc" that actually have the id "ValidateDocumentAge"
